docs/source/man.rst

```diff
@@ -211,9 +211,9 @@ When you configure a scan, it contains references to one or more sources, includ
 Creating and Editing Sources
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To create a source, supply the type of source with the ``type`` option, one or more host names or IP addresses to connect to with the ``--hosts`` option, and the credentials needed to access those systems with the ``--cred`` option. The ``qpc source`` command allows multiple entries for the ``hosts`` and ``cred`` options. Therefore, a single source can access a collection of servers and subnets as needed to create an accurate and complete scan.
+To create a source, supply the type of source with the ``type`` option, one or more host names or IP addresses to connect to with the ``--hosts`` option, and the credentials needed to access those systems with the ``--cred`` option. When creating a ``network type`` source, to specify a host or range of hosts to be excluded from the scan use the ``--exclude-hosts`` option. The ``qpc source`` command allows multiple entries for the ``hosts`` and ``cred`` options. Therefore, a single source can access a collection of servers and subnets as needed to create an accurate and complete scan.
```

Comment:
Since this is an optional argument, I don't think that we should include it in this paragraph because it seems like the purpose of this paragraph is giving the essentials to creating a source. 

If we do want to include this I would change it from ``network type`` to ``network``,  and make it clear that this is an optional flag (not required like type/cred/hosts). I would also add ``exclude_hosts`` in the sentence below where it says ```allows multiple entries for the hosts and cred options``` since `exclude_hosts` also allows multiple entries.